HANDBOOK.md

```diff
@@ -191,31 +193,28 @@ It is your choice to build the barriers first or the map first. For this example
+* Follow the instructions given to you in chat and select the area your map is in, just like how you would select the area with WorldEdit. You can either punch a node to select or stand where you want to select and run the following command:
   ```
   /ctf_map here
   ```
   ![image](https://github.com/CTF-handbooks/map-maker-handbook/assets/88883098/bdca51e0-a439-4bf3-a65a-75530f2f5d2d)
   
-  Make sure you selected your entire map and make sure you did not select any extra space. If you have decided to leave some terrain/structures outside your outer glass barrier, make sure to include them in too.
+  Make sure you selected your entire map and make sure you did not select any extra space. If you have decided to leave some terrain/structures outside your outer glass barrier, include them too.
 ***
 ### 11. Exporting the map
 * After selecting your map area, run `/ctf_map editor` in chat again. This will open the Map Editor.
-  ```
-  /ctf_map editor
-  ```
-* Check `Map Enabled` unless you don't want to make the map available for play.
+* Check `Map Enabled` unless you don't want to make the map available for play. If you are making the map for addition to the official server, you will check this.
   
   ![image](https://github.com/CTF-handbooks/map-maker-handbook/assets/88883098/a1bfc7bb-0b84-4cea-b41c-b15668c15f55)
 
-* In the license field, add the license of your map. Every map must have a license. The official CTF server will only accept free culture licenses like `CC BY-SA 4.0`. Please note that some CC licenses such as `CC BY-ND` are not free. You can find a list of CC licenses [here](https://creativecommons.org/share-your-work/cclicenses/).
+* In the license field, add the license of your map. Every map must have a license. The official CTF server will only accept free culture licenses like `CC BY-SA 4.0`. Please note that some CC licenses, such as `CC BY-ND` are not free. You can find a list of CC licenses [here](https://creativecommons.org/share-your-work/cclicenses/).
 
   ![image](https://github.com/CTF-handbooks/map-maker-handbook/assets/88883098/e3e0a9e8-8968-423a-8b54-3288f303d0d9)
 
   From the ctf_map README:
-  >You can append any attribution you need to give to the `license` field (For example: If you modified someone's map or used one of their builds you'd list their name and what map/build of theirs that you modified/used). If you want to give more information, you can use the `Other Info` field.
+  > You can append any attribution you need to give to the `license` field (For example: If you modified someone's map or used one of their builds, you'd list their name and what map/build of theirs that you modified/used). If you want to give more information, you can use the `Other Info` field.
```

Comment:
```suggestion
  > You can append any attribution you need to give to the `license` field (For example: If you modified someone's map or used one of their builds you'd list their name and what map/build of theirs that you modified/used). If you want to give more information, you can use the `Other Info` field.
```
This is a straight out quote from the ctf_map README, so I'm not editing.

Reply:
Ah right, it is from the quote.